cypress/integration/createBoard.spec.js

```diff
@@ -15,11 +15,20 @@ describe("loginUser", () =>{
         cy.get(login.emailInputField).type(data.user.email);
         cy.get(login.passwordInputField).type(data.user.password);
         cy.get(login.loginButton).click(); 
-        cy.wait(3000); 
+        
+        cy.get(login.loginButton).should("not.exist");
+    })
+
+    afterEach('logOut', () => {
+        cy.get(login.profileButton).should("be.visible").click();
+        cy.get(login.settingsButton).click();
+        cy.get(login.logOutButton).click();
+
+        cy.get(login.loginButton).should('be.visible').and("contain", "Log In");
     })
 
     it("cancelCreatingBoard", () => {
```

Comment:
These are not very descriptive - you should strive for a more clear description of the tests, just as you would when you talk to a stakeholder/PO/whoever defines certain functionality. For example: 

`it("I can cancel board creation", () => {})`

This truly distinguishes excellent tests from "meh" tests :)